src/resolve/resolved-dns-cache.h

```diff
@@ -31,4 +32,4 @@ bool dns_cache_is_empty(DnsCache *cache);
 
 unsigned dns_cache_size(DnsCache *cache);
 
-int dns_cache_export_shared_to_packet(DnsCache *cache, DnsPacket *p);
+int dns_cache_export_shared_to_packet(DnsCache *cache, DnsPacket *p);
```

Comment:
please keep the trailing newline in place

Reply:
Done